frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -601,13 +571,18 @@ const ManagePolicyPage = ({
     ? teamPoliciesError
     : globalPoliciesError;
 
-  const policyResults = isAnyTeamSelected ? !!teamPolicies : !!globalPolicies;
+  const policyResults = isAnyTeamSelected
+    ? teamPolicies && teamPolicies.length > 0
+    : globalPolicies && globalPolicies.length > 0;
```

Comment:
This was the bug. Because teamPolicies and globalPolicies is now set to `[]` for API returning `{}`, this now was evaluating to true instead of false. Updated it to check the length of the array